.github/workflows/continuous-integration.yaml

```diff
@@ -551,7 +552,8 @@ jobs:
         id: build-docs
         if: |
           (
-            steps.changed-files-in-pr.outcome == 'skipped' 
+            (fromJSON(env.IS_RELEASE) && fromJSON(env.IS_REF_BUILD))
+            || steps.changed-files-in-pr.outcome == 'skipped' 
```

Comment:
Why is this needed?

Reply:
it was already here before (see above), it just moved (but basically we avoid building docs in PRs if no documentation files changed)